_data/publications.json

```diff
@@ -0,0 +1,806 @@
+{
+  "publications": [
```

Comment:
I think it would make things a bit more manageable if we had one JSON file for each of the 4 (5?) "enumerations".

(But if you prefer to do that as a follow-up PR, that also works.)

Reply:
Yeah, we can do that later I think. Having things in one file will make it easier to refactor if we need to change data format.